File: OEM/Domain/ComplementaryTasks/ComplementaryTaskDto.cs
using System;

namespace OEMAPI.Domain.ComplementaryTasks
{
    public class ComplementaryTaskDto
    {
        public Guid Id { get; set; }
        public string CategoryId { get; set; }
        public string VveId { get; set; }
        public string ResponsibleTeam { get; set; }
        public DateTime StartTime { get; set; }
        public DateTime? EndTime { get; set; }
        public string Status { get; set; }
    }
}

File: OEM/src/dto/ComplementaryTaskDTO.ts
export interface ComplementaryTaskDTO {
  id: string;
  vesselVisitExecutionId: string;
  categoryId: string;
  responsibleTeam: string;
  startTime: string;
  endTime?: string;
  status: string;
  executionMode?: string;
}

export interface CreateComplementaryTaskDTO {
  vesselVisitExecutionId: string;
  categoryId: string;
  responsibleTeam: string;
  startTime: string;
  endTime?: string;
  status: string;
  executionMode?: 'PARALLEL' | 'SUSPEND';
}

export interface UpdateComplementaryTaskDTO {
  responsibleTeam?: string;
  startTime?: string;
  endTime?: string;
  status?: string;
}